### PR TITLE
Added info about modules options

### PR DESCRIPTION
**Pull request by kayrus**

### CODE REVIEW

File: os/other-settings.md [hunk 1]
```diff
@@ -33,10 +33,31 @@ coreos:
       command: restart
 ```
 
+### Loading Kernel Modules with Options
+
+This example will load `dummy` module with `numdummies=5` option:
```
Comment (joshix) [inline os/other-settings.md]: This example cloud-config excerpt loads the `dummy` network interface module with configuration specifying the number of interfaces the module should create when loaded (`numdummies=5`):

File: os/other-settings.md [hunk 2]
```diff
@@ -33,10 +33,31 @@ coreos:
       command: restart
 ```
 
+### Loading Kernel Modules with Options
```
Comment (joshix) [inline os/other-settings.md]: ### Loading Kernel Modules with Configuration

File: os/other-settings.md [hunk 3]
```diff
@@ -33,10 +33,31 @@ coreos:
...
+  - path: /etc/modules-load.d/dummy.conf
+    content: dummy
+
+coreos:
+  units:
+    - name: systemd-modules-load.service
+      command: restart
+```
+
+As a result you will get five dummy network interfaces.
```
Comment (joshix) [inline os/other-settings.md]: After this cloud-config is processed, five `dummy` interfaces are added to the network stack.

File: os/other-settings.md (hunk 2 above)
Comment (kayrus) [inline os/other-settings.md]: I would argue this this. Linux world knows module options, not module configuration.

**Comment by kayrus** [comment]
@joshix Fixed

File: os/other-settings.md (hunk 2 above)
Comment (joshix) [inline os/other-settings.md]: Fair enough, you may be right. Stet.

File: os/other-settings.md [hunk 4]
```diff
@@ -33,10 +33,31 @@ coreos:
       command: restart
 ```
 
+### Loading Kernel Modules with Options
+
+This example cloud-config excerpt loads the `dummy` network interface module with option specifying the number of interfaces the module should create when loaded (`numdummies=5`):
```
Comment (joshix) [inline os/other-settings.md]: s/option/options/

Comment (kayrus) [inline os/other-settings.md]: there is only one option - `numdummies=5`

Comment (joshix) [inline os/other-settings.md]: Heh. True enough. So:

"...interface module**,** with **an** option specifying..."

**Pull request merged**